src/status_im/ui/screens/chat/styles/message/message.cljs

```diff
@@ -29,11 +29,26 @@
 (def message-timestamp
   {:font-size  10})
 
-(defn message-timestamp-placeholder
-  []
-  (merge message-timestamp {:opacity 0 :color "rgba(0,0,0,0)"}))
+(defn message-timestamp-wrapper [{:keys [last-in-group? outgoing group-chat]}]
+  {
+   :justify-content :center
+   (if outgoing :margin-right :margin-left) 12
+   :margin-top (if (and last-in-group?
```

Comment:
A comment as to why we need different margins for different cases will be helpful.